src/app.js

```diff
@@ -147,9 +147,9 @@ export default function(app) {
       var value = data[name]
       var oldValue = oldData[name]
 
-      if (name === "onupdate") {
+      if (name == "onupdate") {
         value(element)
-      } else if (value !== oldValue || value !== element[name]) {
+      } else if (value != oldValue || value != element[name]) {
```

Comment:
This one is what I'm worried about the most. No yet 100% familiar with the source or intended behvaiour so can't say for sure what can happen here. We should add a test for it anyway.